<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitKeeper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>4</number>
         </property>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="handleWidth">
              <number>4</number>
             </property>
             <widget class="QTableView" name="filesTableView">
              <property name="styleSheet">
               <string notr="true">selection-background-color: rgb(185, 162, 255);</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::NoPen</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
             <widget class="QTableView" name="stagedTableView">
              <property name="editTriggers">
               <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="splitter_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="diffToolbar" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0,0">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QToolButton" name="prevFileBtn">
                    <property name="text">
                     <string>prevFile</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="nextFileBtn">
                    <property name="text">
                     <string>nextFile</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QToolButton" name="prevFileChangeBtn">
                    <property name="text">
                     <string>Prev</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="nextFileChangeBtn">
                    <property name="text">
                     <string>Next</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="diffViewerPanel" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="CodeEditor" name="originalFileEdit">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="documentTitle">
                     <string notr="true"/>
                    </property>
                    <property name="undoRedoEnabled">
                     <bool>false</bool>
                    </property>
                    <property name="lineWrapMode">
                     <enum>QPlainTextEdit::NoWrap</enum>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="plainText">
                     <string notr="true"/>
                    </property>
                    <property name="backgroundVisible">
                     <bool>false</bool>
                    </property>
                    <property name="centerOnScroll">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="diffActionsWidget" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="autoFillBackground">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="CodeEditor" name="currentFileEdit">
                    <property name="documentTitle">
                     <string notr="true"/>
                    </property>
                    <property name="undoRedoEnabled">
                     <bool>false</bool>
                    </property>
                    <property name="lineWrapMode">
                     <enum>QPlainTextEdit::NoWrap</enum>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="plainText">
                     <string notr="true"/>
                    </property>
                    <property name="backgroundVisible">
                     <bool>false</bool>
                    </property>
                    <property name="centerOnScroll">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                 <zorder>currentFileEdit</zorder>
                 <zorder>originalFileEdit</zorder>
                 <zorder>diffActionsWidget</zorder>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tabWidgetPage1">
               <attribute name="title">
                <string>Commit</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QTextEdit" name="commitMessageEdit">
                  <property name="acceptRichText">
                   <bool>false</bool>
                  </property>
                  <property name="placeholderText">
                   <string>Commit message...</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
                  <item>
                   <widget class="QCheckBox" name="amendCheckBox">
                    <property name="text">
                     <string>Amend</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btnCommit">
                    <property name="text">
                     <string>Commit</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Output</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QTextEdit" name="logMessage">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(206, 206, 206);</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btnClearLog">
                    <property name="text">
                     <string>Clear</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="actionOpenSettings"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Изменения</string>
    </property>
    <addaction name="nextFileChangeAction"/>
    <addaction name="prevFileChangeAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStatus"/>
   <addaction name="actionLog"/>
   <addaction name="actionCommit"/>
   <addaction name="actionFetch"/>
   <addaction name="actionPull"/>
   <addaction name="actionPush"/>
  </widget>
  <action name="actionStatus">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionCommit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </action>
  <action name="actionFetch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fetch</string>
   </property>
  </action>
  <action name="actionPull">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pull</string>
   </property>
  </action>
  <action name="actionPush">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Push</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actionOpenSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Клонировать</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="nextFileChangeAction">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next change</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="prevFileChangeAction">
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="toolTip">
    <string>Previous changes</string>
   </property>
   <property name="shortcut">
    <string>Shift+F6</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">widgets/code_editor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
